CONTRIBUTING.md

```diff
@@ -11,7 +11,7 @@ into a single commit. Please provide ample explanation in the commit message.
 Installation
 ------------
 
-Testing the Parse PHP SDK requires having a working Parse Server instance to run against. 
+Testing the Parse PHP SDK requires having a working Parse Server instance to run against.
```

Comment:
I've noticed a lot of line endings were adjusted throughout, did you end up normalizing them?